azuredeploy.json

```diff
@@ -24,12 +24,68 @@
             "type": "bool"
         },
         "azureBackupSwitch": {
-            "defaultValue": false,
+            "defaultValue": true,
             "metadata": {
                 "description": "Switch to configure AzureBackup and enlist VM's"
             },
             "type": "bool"
         },
+        "azureSearchSwitch": {
+            "defaultValue": false,
+            "metadata": {
+                "description": "Switch to configure Azure Search"
+            },
+            "type": "bool"
+        },
+        "azureSearchSku": {
+            "defaultValue": "basic",
+	    "allowedValues": [
```

Comment:
Please be mindful with indentation. There are many places indentation is broken. Please do fix all of them.